Controller/CoreController.php

```diff
@@ -129,10 +124,27 @@ public function searchAction(Request $request)
         }
 
         return $this->render($this->container->get('sonata.admin.pool')->getTemplate('search'), array(
-            'base_template' => $this->getBaseTemplate($request),
+            'base_template' => $this->getBaseTemplate(),
             'admin_pool'    => $this->container->get('sonata.admin.pool'),
             'query'         => $request->get('q'),
             'groups'        => $this->getAdminPool()->getDashboardGroups(),
         ));
     }
+
+    /**
+     * The purpose of this method it to be BC with symfony 2.
+     *
+     * @deprecated If you use symfony 3, consider using a Request action
+     *             argument.
```

Comment:
i would say: "Use the Request action argument. This method will be removed in SoantaAdminBundle 3.0 and the action methods adjusted."